app/javascript/psulib_blacklight/javascripts/availability.js

```diff
@@ -22,142 +28,239 @@ $(document).on('turbolinks:load', function() {
 /**
  * Load real time holdings and availability info from Sirsi Web Services
  */
-function loadAvailability(locations, item_types) {
-    // Load Sirsi locations
-    var allLocations = locations.locations;
-    var allLibraries = locations.libraries;
-    var illiadLocations = locations.request_via_ill;
-    var allItemTypes = item_types.item_types;
+function loadAvailability() {
     var titleIDs = [];
 
     // Get the catkeys
-    $('.availability').each(function() {
+    $('.availability').each(function () {
```

Comment:
```suggestion
    $('.availability').each(function() {
```

Formatting, but I don't think we need a space between `function` and `()`. We should implement a linter.